pattern-lab/source/_patterns/00-atoms/02-text/01-paragraph.twig

```diff
@@ -16,7 +16,7 @@
 
 <p >
   {% block paragraph_content %}
-    {{ paragraph_content }}
+    {{ paragraph_content|striptags }}
```

Comment:
a paragraph can have tags eg: a, strong etc. so this striptags should not be here.

Reply:
Removed it